copilot-theorem/src/Copilot/Theorem/What4/Translate.hs

```diff
@@ -497,11 +523,11 @@ translateOp1 origExpr sym op xe = case (op, xe) of
                        e_neg <- WI.bvSlt sym e zero
                        neg_e <- WI.bvSub sym zero e
                        WI.bvIte sym e_neg neg_e e
-          fpAbs :: FPOp1 sym fpp
-          fpAbs e = do zero <- WI.floatLit sym knownRepr bfPosZero
-                       e_neg <- WI.floatLt sym e zero
-                       neg_e <- WI.floatSub sym fpRM zero e
-                       WI.floatIte sym e_neg neg_e e
+          fpAbs :: forall fi . FPOp1 sym fi
```

Comment:
We should double check that these "constructed" operations actually have the desired semantics.  This stuff can get very tricky.

Reply:
I've left a reminder to do this in https://github.com/GaloisInc/copilot-verifier/issues/3#issuecomment-944643076.